src/assets/styles/theme.scss

```diff
@@ -73,3 +73,7 @@ $px-15: 0.9375rem;
 $px-16: 1.0000rem;
 $px-22: 1.375rem;
 $px-25: 1.5625rem;
+
+// rules and memorandum style
+$app-lt-gray: #38598a;
+$px-17: 17px;
```

Comment:
Please move these near their brothers and sisters.

I'm OK with an end-if-line comment saying it's only for rules and memorandum if you want.

Reply:
I just switched it to app-dk-blue

Reply:
OK. I just wanted to keep related variables together.

Reply:
Fix confirmed.